apparmor.d/groups/network/nm-dispatcher

```diff
@@ -57,13 +60,53 @@ profile nm-dispatcher @{exec_path} flags=(attach_disconnected) {
   @{run}/systemd/notify rw,
   @{run}/tlp/{,*} rw,
   @{run}/chrony-dhcp/ rw,
+  @{run}/ntp.conf.dhcp rw,
 
   @{sys}/class/net/ r,
 
   owner @{PROC}/@{pid}/fd/ r,
 
   /dev/tty rw,
 
+  profile systemctl {
+    include <abstractions/base>
+    include <abstractions/systemd-common>
+
+    @{bin}/systemctl mr,
+
+    owner / r,
+
+    owner @{etc_ro}/ r,
+    owner @{etc_ro}/systemd/ r,
+    owner @{etc_ro}/systemd/system/ r,
+    owner @{etc_ro}/systemd/system/ntp.service r,
```

Comment:
No owner for `/etc` and `/`. That make the profile uselessly verbose (ultimately almost everything in a privileged program could be prefixed by a `owner`)